packages/core/stories/patterns/app-header/app-header.stories.tsx

```diff
@@ -0,0 +1,218 @@
+import { FC, HTMLAttributes, ReactNode, useState } from "react";
+import {
+  Button,
+  FlexItem,
+  FlexLayout,
+  StackLayout,
+  BorderLayout,
+  BorderItem,
+  useResponsiveProp,
+  Text,
+} from "@salt-ds/core";
+import { Drawer, NavigationItem } from "@salt-ds/lab";
+import {
+  SymphonyIcon,
+  StackoverflowIcon,
+  GithubIcon,
+  MenuIcon,
+  CloseIcon,
+} from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+
+export default {
+  title: "Patterns/App Header",
+} as Meta;
+
+export const AppHeader = () => {
+  interface AppHeaderProps extends HTMLAttributes<HTMLElement> {
+    /**
+     * Items included in the App Header navigation
+     */
+    items?: string[];
+    /**
+     * Utilities included in the App Header
+     */
+    utilities?: { icon: ReactNode; label: string; key: string }[];
+  }
+
+  const AppHeader: FC<AppHeaderProps> = ({ items, utilities }) => {
+    const [drawerOpen, setDrawerOpen] = useState(false);
+    const [active, setActive] = useState(items?.[0]);
+    const isMobile = useResponsiveProp({ xs: true, sm: false }, false);
+
+    const handleDrawerToggle = () => {
+      setDrawerOpen(!drawerOpen);
+    };
+
+    return (
+      <header>
+        {isMobile ? (
+          // Mobile Header
+          <>
+            <StackLayout
+              direction="row"
+              style={{
```

Comment:
A general note, I'm concerned with all the inline styling but I'm not sure if we have a better option at the moment.

Reply:
yeah I guess we would recommend users to separate it. I also made a [codesandbox](https://codesandbox.io/p/sandbox/app-header-9l6nky?file=%2Fsrc%2Fstyles.css) it's a bit out of sync but it will be a more realistic example.

Reply:
yeah I figured it's not ideal, but nowhere else to put it